enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,439 @@
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+N/A
+
+#### Tech Preview -> GA
+
+See Release Plan above.
+
+#### Removing a deprecated feature
+
+N/A, as this is a new feature that does not supersede an existing one.
+
+### Upgrade / Downgrade Strategy
+
+The relevant upgrade path is from WMCO 9.y.z in OCP 4.14 to WMCO 10.15.0 in OCP 4.15. There will be no changes to the
+current WMCO upgrade strategy. Once customers are on WMCO 10.15.0, they can [convert their connected cluster to a
+disconnected one](https://docs.openshift.com/container-platform/4.14/post_installation_configuration/connected-to-disconnected.html#connected-to-disconnected-config-registry_connected-to-disconnected)
+and the Windows nodes will be automatically updated by the operator to use the mirror settings to pull required images.
+
+When deconfiguring Windows nodes, mirror settings will be cleared from the Windows instances. This involves undoing some node
+config steps, which will be taken care of as part of removing the `containerd` Windows service.
+This scenario will occur when upgrading both BYOH and Machine-backed Windows instances.
+
+Downgrades are generally [not supported by OLM](https://github.com/operator-framework/operator-lifecycle-manager/issues/1177),
+which manages WMCO. In case of breaking changes, please see the
+[WMCO Upgrades](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator-upgrades.md#risks-and-mitigations)
+enhancement document for guidance.
+
+### Version Skew Strategy
+
+N/A. There will be no version skew since this work is all within 1 product sub-component (WMCO). 
+The official Red Hat operator in OCP 4.15, WMCO version 10.15.0, will support Windows containers in disconnected
+environments. Releases 9.y.z and below will not have support disconnected support.
+
+## Implementation History
+
+The implementation history can be tracked by following the associated work items in Jira and source code improvements in
+the WMCO Github repo.
+
+## Alternatives & Justification
+ 
+For disconnected support, we could make the kubelet pause image location configurable. This is a nice to have, but may 
+be out of scope of disconnected support as it could require a larger undertaking to make the containerd config file
+configurable.
+
+There are a few alternatives that are no longer viable.
+Pre-pulling images, where user creates a Windows image with the pause image and other required images baked in, is not
+possible anymore as we install and manage the container runtime.
+Also, users cannot configure containerd's registries manually since WMCO will overwrite changes to the containerd
+config file if there is some unrelated event that triggers node reconciliation.
+
+## Open Questions
+
+* Would it work "out-of-the-box" today if images were pre-loaded with specific tags into the OpenShift internal registry,
```

Comment:
I don't understand this open question or it's implications

Reply:
It served mostly as an info dump of a workaround to get WMCO working in a disconnected env that required the least possible dev work. I will remove this